includes/Modules/AdSense/Web_Tag.php

```diff
@@ -54,8 +54,9 @@ protected function render() {
 		// because it is required for account verification.
 
 		$adsense_script_src = sprintf(
-			'https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=%s',
-			esc_attr( $this->tag_id )
+			'https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=%s&host=%s',
+			esc_attr( $this->tag_id ), // Site owner's web property code
+			'ca-host-pub-2644536267352236' // SiteKit's web property code
```

Comment:
This same value is used in the meta tag as well:
https://github.com/google/site-kit-wp/blob/ca7fafed0dff8a4756b783872428ce864dc8c337/includes/Modules/AdSense.php#L827

@felixarntz Perhaps we should extract it to a constant on the `Modules\AdSense` class?

Reply:
@aaemnnosttv We could, but don't really think it's necessary.